docs/source/architecture/poet.rst

```diff
@@ -0,0 +1,620 @@
+**********************
+PoET 1.0 Specification
+**********************
+
+Introduction
+============
+
+The Proof-of-Elapsed-Time or PoET Consensus method offers a solution to the
+Byzantine Generals Problem that utilizes a “trusted execution environment” to
+improve on the efficiency of present solutions such as Proof-of-Work. The
+following presentation assumes the use of Intel SGX as the trusted execution
```

Comment:
Before the last sentence please insert:
The initial release of PoET was fully abstracted so as to be portable to any trusted execution environment.